CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### 0.26.10 - 2016-01-14
```

Comment:
You shouldn't need to manually edit the changelog or the version number. When you do a release. you pass int eh change log details - see readme on releasing (unless this has changed)